spec/order_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'order'
+require 'menu'
+
+describe Order do
+  let(:menu_list) { {
+    :caesar_salad => 4,
+    :gazpacho => 3,
+    :tartiflette => 6,
+    :butternut_squash_risotto => 6,
+    :roasted_vegetables => 6,
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)